port_drayage_plugin/config/params.yml

```diff
@@ -1,5 +1,4 @@
 stop_speed_epsilon: 1.0
-cmv_id: "123" # Test value; CARMA Streets expects cmv_id to be an unsigned long (32-bit unsigned integer)
```

Comment:
CARMA Streets expects a 32bit unsigned int. That limits the id string to at most 4 characters. All of our vehicle Ids are longer than 4 characters. Either some use case specific encoding needs to be happening or the message updated to support a larger value.

Reply:
This comment was from an earlier design decision that required cmv_id to be set to a 32-bit unsigned integer. Later in testing, it was detemrined that the cmv_id should match the vehicle's vehicle_id parameter. V2XHub/CARMA Streets has already been updated to expect and treat the cmv_id value as a string in received MobilityOperation messages for Port Drayage.